src/test/common/services/sendEmail.test.js

```diff
@@ -26,9 +26,12 @@ describe('Send Email Service', () => {
     sinon.restore();
   });
 
+  // For the sake of code coverage, have this rewire the logger for production...
   it('should throw an error if no API key set', () => {
+    process.env.NODE_ENV = 'production';
```

Comment:
This does not fully test what happens when the logger is in 'production'. Purely for code coverage purposes.